README.md

```diff
@@ -152,9 +152,14 @@ If you're working on PR for this project, create a feature branch off of `main`.
 This repository follows the [Samvera Community Code of Conduct](https://samvera.atlassian.net/wiki/spaces/samvera/pages/405212316/Code+of+Conduct) and [language recommendations](https://github.com/samvera/maintenance/blob/main/templates/CONTRIBUTING.md#language).  Please ***do not*** create a branch called `master` for this repository or as part of your pull request; the branch will either need to be removed or renamed before it can be considered for inclusion in the code base and history of this repository.
 
 ## Compatibility
-
-- Ruby 2.5 or the latest 2.4 version is recommended. Later versions may also work.
-- Rails 5 is required. We recommend the latest Rails 5.2 release.
+This code works with the latest versions of:
+ - Rails 6.1 and 6.0.
+   - Rails 5 also works with Ruby versions older than 3.
+ - Ruby 3.0 (except with Rails 5), 2.6, and 2.5.
```

Comment:
The caveats is a good section. Otherwise, let's just say:

This gem is tested/supported Rails 5.0-6.1 and ruby with Ruby 2.5-3.1 (so long as that ruby version is supported by the Rails version!).

Reply:
Fixed.